client-js/app/elements/labrad-registry.html

```diff
@@ -6,40 +6,188 @@
       display: block;
     }
     
-    .list {
+    #reg-area {
       padding-top: 12px;
       background-color: white;
       display: inline-block;
       @apply(--shadow-elevation-2dp);
+    }
+    .iron-selected {
+      background-color: #ddd;
+    } 
+    .iron-selector {
+      width: 100%;
+      display: block;
+      border: 1px solid #ccc;
+      border-top: none;
+    }  
+    .reg-list { 
+      padding-top: 12px;
+      display: table; 
+      border: 1px solid;
+    }
+    .row { 
+      display: table-row; 
+      padding: 12px 24px;
+      width: 100%;
+    }
+
+    .head-item { 
+      display: table-cell; 
+      padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+      width: 100%;
+    } 
+    .head-grabber { 
+      display: table-cell; 
+      *padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+    } 
+
+    tbody {
+      display:table-header-group;
     }    
+
+    .table-header {
+      @apply(--paper-font-body1);
+      color: var(--secondary-text-color);
+      *text-align: middle;
+      border-bottom: 1px solid;
+      border-bottom-color: #e5e5e5;
+      display:table-header-group;
+
+    } 
+
+    paper-dialog.size-position {
+      position: fixed;
+      top: 16p
+      right: 16px;
+    }
+    paper-dialog.size-position {
+      width: 300px;
+      height: 300px;
+    }
+    menu-button.done {
+      color: #0F9D58;
+    }
   </style>
   <template>
-  <div class="list short">
-    <table>
-      <thead>
-        <th>Key/Dir Name</th>
-        <th>Key Value</th>
-      </thead>
-      <tbody>
-        <template is="dom-repeat" items="{{dirs}}">
-          <tr>
-            <td>
-              <a is="app-link" path="{{item.url}}" href="{{item.url}}">
-               <dir-item kind="folder">{{item.name}}</dir-item>
-              </a>
-            </td>
-            <td></td>
-          </tr>
-        </template>
-        <template is="dom-repeat" items="{{keys}}">
-          <tr>
-            <td><dir-item kind="key">{{item.name}}</dir-item></td>
-            <td><span class="paper-font-body1">{{item.value}}</span></td>
-          </tr>
-        </template>
-      </tbody>
-    </table>
+    <div id="reg-menu"> 
+    <section onclick="clickHandler(event)">
```

Comment:
it looks like if you use `on-click` (a special polymer attribute) instead of `onclick` (a standard DOM attribute) then polymer will wire this up to call an instance method on the element itself, so you could avoid having `clickHandler` be a global function. (https://www.polymer-project.org/1.0/docs/devguide/events.html#annotated-listeners)